allocation/allocation_runner.py

```diff
@@ -4,9 +4,14 @@
 from allocation.allocation_solver import AllocationSolver
 from allocation.models import AllocationRequest
 from applications.allocation_result_mapper import AllocationResultMapper
+from applications.models import ApplicationEventScheduleResult
 
 
 def start_allocation(allocation_request: AllocationRequest):
+    ApplicationEventScheduleResult.objects.filter(
+        accepted=False,
+        application_event_schedule__application_event__application__application_round=allocation_request.application_round,  # noqa: E501
+    ).delete()
```

Comment:
I think it would be nice if the deletion of the schedule result would be in mapper class?

Reply:
Yeah, i think that would be a better place, so those get deleted only when we actually get a solution.

Reply:
Although then we need to add some logic for skipping the accepted events in AllocationDataBuilder but that's probably fine.

Reply:
Never mind, i'm stupid it's already handled in ApplicationEvent#get_not_scheduled_occurrences ...